### ISSUE TITLE
Vagan Dagger "Hits can't be Evaded"

### ISSUE THREAD

**Issue by MattChamberlain** [Openarl/PathOfBuilding#342]
The only dagger base that has the Vagan mod "Hits can't be Evaded" is Royal Skean even though you can get that modifier on other bases.

**Comment by Openarl** [comment]
Of course, but the intent of the rare templates isn't to cover every possible base (there would be far too many of them otherwise), but rather to give you a way of approximating a build's final gear setup, and the difference between bases isn't generally large enough to justify having multiple templates for one weapon type.
